src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -0,0 +1,499 @@
+            description: 'I certify that the individual listed below is my Institutional Signing official',
+            title: '1.6 Institutional Signing Official',
+            validators: [FormValidators.REQUIRED],
+            ariaLevel: ariaLevel + 1,
+            defaultValue: signingOfficial,
+            onChange: ({key: name, value}) => {
+              const formattedValue = isNil(value) ? '' : formatSOString(value.displayName, value.email);
+              formFieldChange({name, value: formattedValue});
+            },
+            selectOptions: allSigningOfficials,
+            selectConfig: {
+              placeholder: 'Signing Official',
+              getOptionLabel: (option) => formatSOString(option.displayName, option.email), //formats labels on dropdown
+              getNewOptionData: (inputValue) => { //formats user input into object for use within Creatable
+                return { displayName: inputValue };
+              },
+              getOptionValue: (option) => { //value formatter for options, attr used to ensure empty strings are treated as undefined
+                if(isNil(option) || isEmpty(option.displayName)) {
+                  return null;
+                }
+                return option;
+              }
+            },
+          }),
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          h(FormField, {
+            id: 'itDirector',
+            description: 'I certify that the individual listed below is my IT Director',
+            placeholder: 'IT Director',
+            title: '1.7 Information Technology (IT) Director',
+            validators: [FormValidators.REQUIRED],
+            ariaLevel: ariaLevel + 1,
+            onChange: ({key: name, value,}) => formFieldChange({name, value}),
+            defaultValue: itDirector
+          })
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          div([
+            h(FormField, {
+              id: 'anvilUse',
+              type: FormFieldTypes.RADIO,
+              title: '1.8 Cloud Use Statement',
+              description: [span([
+                'Will you perform all of your data storage and analysis for this project on the ',
+                a({
+                  rel: 'noopener noreferrer',
+                  href: 'https://anvil.terra.bio/',
+                  target: '_blank'
+                }, ['AnVIL']),
+                '?'
+              ])],
+              options: [
+                { name: 'yes', text: 'Yes' },
+                { name: 'no', text: 'No' }
+              ],
+              validators: [FormValidators.REQUIRED],
+              ariaLevel: ariaLevel + 1,
+              orientation: 'horizontal',
+              onChange: ({key: name, value}) => {
+                const normalizedValue = value && value.selected === 'yes';
+                formFieldChange({name, value: normalizedValue});
+              },
+              defaultValue: { selected: anvilUse ? 'yes' : 'no' }
```

Comment:
by default, this is no when you open the page. however, to get the conditional fields underneath it to render, you need to actually click no. I think by default the radio shouldn't be checked at all (i.e. value should be null by default).